x-pack/plugins/security_solution/public/entity_analytics/components/risk_summary_flyout/risk_summary.tsx

```diff
@@ -182,15 +186,17 @@ const RiskSummaryComponent = <T extends RiskScoreEntity>({
               defaultMessage="View risk contributions"
             />
           ),
-          link: {
-            callback: () => openDetailsPanel(EntityDetailsLeftPanelTab.RISK_INPUTS),
-            tooltip: (
-              <FormattedMessage
-                id="xpack.securitySolution.flyout.entityDetails.showAllRiskInputs"
-                defaultMessage="Show all risk inputs"
-              />
-            ),
-          },
+          link: riskScoreData.loading
```

Comment:
This change fixes a cypress flakiness, it was clicking on the link before `riskScoreData` was loaded.